tests/js/JSONSchemasInterface.tests.ts

```diff
@@ -23,7 +23,9 @@ describe("JSONSchemasInterface", () => {
         expect(schema?.properties?.inSet).to.be.an("object");
 
         if (
+            // @ts-ignore
```

Comment:
Maybe just fail if schema is undefined instead or add it to if?

Reply:
Right. Removed ifs